src/server/test/util/vaidationHelpers.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const { chai, app } = require('../common');
+
+chai.use(chaiHttp);
+
+/**
+ * Sends a POST request to the specified API endpoint with a test payload,
+ * setting the given field to an invalid value or removing it entirely
+ * if undefined, to test validation errors.
+ *
+ * @param field the field name in the payload to invalidate
+ * @param invalidValue the value to assign to the field for testing; if undefined, the field is removed
+ * @param endpoint the API endpoint URL to test (e.g., /api/units/addUnit)
+ * @param basePayload the base valid payload object to clone and modify
+ * @param expectedStatus the expected HTTP status code (default 400 for validation errors)
+ */
+async function testInvalidField({ field, invalidValue, endpoint, basePayload, expectedStatus = 400 }) {
+	const payload = { ...basePayload, [field]: invalidValue };
+	if (invalidValue === undefined) {
+		delete payload[field]; // remove field to simulate required check
+	} else {
+		payload[field] = invalidValue;
+	}	
+	const res = await chai.request(app).post(endpoint).send(payload);
+  	expect(res).to.have.status(expectedStatus);
+}
+
+/**
+ * Validates that the API rejects payloads where minVal is greater than maxVal.
+ *
+ * @param endpoint the API endpoint URL to test (e.g., /api/units/addUnit)
+ * @param basePayload a valid payload object to use as the base for testing
+ */
+async function validateMinMaxRelation({ endpoint, basePayload }) {
```

Comment:
This is in util but specific to these two key values in the payload object. What if the two key value names were passed so it could be used elsewhere? I think there my be other cases that need this? If not, then maybe the name should reflect it is for units.